### PR TITLE
update notebook

### PR DESCRIPTION
**Pull request by ooq**

### CODE REVIEW

File: pywren/pywren-risecamp.ipynb [hunk 1]
```diff
@@ -32,9 +32,17 @@
...
-   "metadata": {},
+   "metadata": {
+    "collapsed": false
+   },
    "outputs": [],
    "source": [
+    "# some libraries that are useful for this tutorial\n",
+    "import sys\n",
+    "if '/Users/qifan/repos/risecamp/pywren/' not in sys.path:\n",
+    "    sys.path.insert(0, '/Users/qifan/repos/risecamp/pywren/')\n",
```
Comment (jey) [inline pywren/pywren-risecamp.ipynb]: Is this debug code that should perhaps be removed?

**Comment by jey** [comment]
Hi Qifan, thanks for the PR. Could you do a `git rebase` of this onto the current master? You should hopefully be able to run something like this:

```
# add the "upstream" remote if you don't already have it
git remote add upstream http://github.com/ucbrise/risecamp.git

# fetch the upstream remote, then rebase the current branch onto latest master
git fetch upstream
git rebase upstream/master

# run tests
pywren ...

# push to github
git push
```

**Comment by ooq** [comment]
@jey I did the rebase with your commands, but how it intertwined with  other commits. Let me know you are able to merge this!

**Comment by jey** [comment]
Hi Qifan! Thanks for giving it a shot. I've merged (and squashed) your changes into this commit: https://github.com/ucbrise/risecamp/commit/7a3bc8292e608f636a1ac26966c7a095ceb368f8

Does this look right to you?

**Pull request closed without merging**